src/core_plugins/kibana/public/visualize/listing/visualize_listing.html

```diff
@@ -116,18 +116,6 @@
               <a class="kuiLink" ng-href="{{ item.url }}">
                 {{ item.title }}
               </a>
-
-              <button
-                class="kuiMicroButton kuiTableRowHoverReveal"
-                ng-click="listingController.edit(item)"
```

Comment:
The edit function still exists in the controller, should probably remove that too